fleet-management-service/src/test/java/cz/fi/muni/pa165/service/VehicleServiceImplTest.java

```diff
@@ -0,0 +1,92 @@
+package cz.fi.muni.pa165.service;
+
+import cz.fi.muni.pa165.dao.interfaces.EmployeeDao;
+import cz.fi.muni.pa165.dao.interfaces.JourneyDao;
+import cz.fi.muni.pa165.dao.interfaces.VehicleDao;
+import cz.fi.muni.pa165.entity.Employee;
+import cz.fi.muni.pa165.entity.Journey;
+import cz.fi.muni.pa165.entity.Vehicle;
+import cz.fi.muni.pa165.enums.Role;
+import org.hibernate.service.spi.ServiceException;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.mockito.runners.MockitoJUnitRunner;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.testng.Assert;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+
+import java.time.Year;
+import java.util.Date;
+import java.util.LinkedList;
+import java.util.List;
+
+import static org.mockito.Matchers.any;
+import static org.mockito.Matchers.anyLong;
+import static org.mockito.Mockito.when;
+
+/**
+ * Created by Michal Balicky on 23/11/2016.
+ */
+@RunWith(MockitoJUnitRunner.class)
+public class VehicleServiceImplTest {
+
+    @Mock
+    private VehicleDao vehicleDao;
+    @Mock
+    private EmployeeDao employeeDao;
+    @Mock
+    private JourneyDao journeyDao;
+    @Autowired
```

Comment:
The reason is that **@Inject** is part of the JSX, while **@Autowired** is a proprietary Spring annotation. They behave exactly the same, so effectively it does not matter which one you use, but just make it consistent across the project and use **@Inject**.